Sources/StreamChatUI/ChatMessageList/ChatMessageListView.swift

```diff
@@ -80,18 +80,28 @@ open class ChatMessageListView: UITableView, Customizable, ComponentsProvider {
     ///   - contentViewClass: The type of message content view.
     ///   - attachmentViewInjectorType: The type of attachment injector.
     ///   - layoutOptions: The message content view layout options.
+    ///   - message: The message data.
     /// - Returns: The cell reuse identifier.
     open func reuseIdentifier(
         contentViewClass: ChatMessageContentView.Type,
         attachmentViewInjectorType: AttachmentViewInjector.Type?,
-        layoutOptions: ChatMessageLayoutOptions
+        layoutOptions: ChatMessageLayoutOptions,
+        message: ChatMessage?
```

Comment:
I assume customers didn't override this frequently, right?

Reply:
Also, it seems we don't have it mentioned in the public docs.

Reply:
Yeah this one should be fine. It will be very rare if someone overriden these one.